ossindex/ossindex.go

```diff
@@ -170,3 +167,17 @@ func AuditPackages(purls []string) ([]types.Coordinate, error) {
 	}
 	return results, nil
 }
+
+func setupRequest(jsonStr []byte) (req *http.Request, err error) {
+	if req, err = http.NewRequest(
+		"POST",
+		getOssIndexUrl(),
+		bytes.NewBuffer(jsonStr)); err != nil {
+		return nil, err
+	}
+
+	req.Header.Set("User-agent", fmt.Sprintf("nancy-client/%s", buildversion.BuildVersion))
```

Comment:
Capitalize the `a`?
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/User-Agent

Reply:
Done in: 2ae546a

Reply:
Something kinda interesting, is `req.Header.Get("User-Agent")` and `req.Header.Get("User-agent")` both work despite changing case. This caught me off guard when I adjusted the test and was like whoa why does it still work. Just figured I'd point this out, as golang seems to ignore case on headers, likely due to people being weirdos and using all kinds of different cases in the wild?